backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -479,7 +478,7 @@ trait StandardAsyncExecutionActor
           |$tmpDirPermissionsAdjustment
           |export _JAVA_OPTIONS=-Djava.io.tmpdir="$$tmpDir"
           |export TMPDIR="$$tmpDir"
-          |export HOME="$home"
+          |export HOME="$$HOME"
```

Comment:
This line feels a little odd to me, but I see that we're just keeping what's working currently.